src/fork/pty/master/mod.rs

```diff
@@ -66,11 +69,18 @@ impl Master {
     /// subsequent calls.
     pub fn ptsname_r(&self, buf: &mut [u8]) -> Result<()> {
         if let Some(fd) = self.pty {
-            // Safety: the vector's memory is valid for the duration
-            // of the call
             unsafe {
```

Comment:
I'd like the main the style that every unsafe block has a saftey comment above it.